src/constants/scope/business-details.js

```diff
@@ -0,0 +1,2 @@
+export const FULL_PERMISSION = 'BUSINESS_DETAILS:FULL_PERMISSION'
+export const SCOPE = [FULL_PERMISSION]
```

Comment:
Assume the intention here is that for now we're only allow users to access and make changes if they have full permission?

Reply:
Yes, users with full-permission are the only ones allowed to access the page.